deployment/chainloop/values.yaml

```diff
@@ -809,12 +812,29 @@ postgresql:
 # Vault server running in development mode --set development=true
 # IMPORTANT: This is not meant to run in production
 
-## Hashicorp Vault chart configuration
-## ref: https://github.com/hashicorp/vault-helm/blob/main/values.yaml
-## @param vault.server.dev.enabled Enable development mode (unsealed, in-memory, insecure)
-## @param vault.server.dev.devRootToken Connection token
+## Bitnami Hashicorp Vault chart configuration
+## ref: https://github.com/bitnami/charts/blob/main/bitnami/vault/values.yaml
+## @param vault.server.args Arguments to pass to the vault server. This is useful for setting the server in development mode
+## @param vault.server.config Configuration for the vault server. Small override of default Bitnami configuration
+## @param vault.server.extraEnvVars[0].name Root token for the vault server
+## @param vault.server.extraEnvVars[0].value The value of the root token. Default: notasecret
+## @param vault.server.extraEnvVars[1].name Address to listen on development mode
+## @param vault.server.extraEnvVars[1].value The address to listen on. Default: [::]:8200
 vault:
   server:
-    dev:
-      enabled: true
-      devRootToken: "notapassword"
+    args: [
```

Comment:
Could you add a comment saying that this is only used when enabled `development` mode in the chart? 

Thanks!

Reply:
Sure, I'll add it to the top of the `vault` key!